presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -43,6 +52,16 @@
  */
 public class RubixInitializer
 {
+    private static final RetryDriver DEFAULT_COORDINATOR_RETRY_DRIVER = retry()
```

Comment:
rename to `WAIT_FOR_COORDINATOR_RETRY_DRIVER`.

edit; skip this. You allow for overriding it in followup commit